dbtwiz/source/create.py

```diff
@@ -322,8 +323,7 @@ def write_source_file(
     with open(source_file, "w", encoding="utf-8") as file:
         ruamel_yaml.dump(data, file)
 
-    # FIXME: Make editor user configurable with 'code' as default
-    os.system(f"code {source_file}")
+    os.system(f"{user_config().editor} {source_file}")
```

Comment:
Might there be some editors that don't support this type of cli arguments for opening a file?
Or should we list some suggestions that we know work like this in the documentation?

Reply:
Maybe rather than having `user_config().editor`, we should customize how it works for specific editors.
E.g. for VS Code, the user config value could be `vscode`, while we specifically define that for vscode, the cli argument is `code <file path>`, and similarly for other known editors.

Alternatively, we need to be very specific about what `user_config().editor` is and isn't. It is specifically the cli command for opening a file in an editor, and needs to support just providing a path without other arguments.

Reply:
There could be individual ways to use even the same editor. For instance, I use _Emacs_ but call it with `emacsclient -n` to open the file in a running instance.

I think a good solution would be to specify it as a command with `{}` where you want the file to be opened to be inserted, and make this clear in the docs.